hellomama_registration/utils.py

```diff
@@ -108,6 +108,18 @@ def get_identity_address(identity):
     return identity_store_client.get_identity_address(identity)
 
 
+def get_address_from_identity(identity):
+    last_address = None
+    for address, detail in identity['details'].get(
+            'addresses', {}).get('msisdn', {}).items():
+        if detail.get('optedout', False) is True:
+            pass
+        if detail.get('default', False) is True:
+            return address
+        last_address = address
+    return last_address
```

Comment:
Instead of using this, we could use the newer `/identities/{uuid}/addresses/{address_type}` endpoint, with the `deafult=true` parameter.

Reply:
I do it this way to avoid doing another call to the identity store, before calling this function I retrieve the whole identity to get the operator id to know where to send the notification.